packages/flutter/test/widgets/autocomplete_test.dart

```diff
@@ -3208,4 +3208,44 @@ void main() {
     expect(lastOptions, altEleOptions);
     expect(find.byKey(optionsKey), findsOneWidget);
   });
+
+  testWidgets('optionsLayerLink: open upward', (WidgetTester tester) async {
+    final LayerLink optionsLayerLink = LayerLink();
+    await tester.pumpWidget(
+      MaterialApp(
+        home: Scaffold(
+          body: Center(
+            child: CompositedTransformTarget(
```

Comment:
Does this example actually produce a different position for the options than if you didn't use optionsLayerLink?